radio/src/telemetry/multi.cpp

```diff
@@ -471,19 +497,14 @@ uint16_t MultiModuleSyncStatus::getAdjustedRefreshRate()
     return rate;
 }
 
-
-static void prependSpaces(char * buf, int val)
+char * prependSpaces(char * buf, int val)
```

Comment:
there is at least a division to remove in this code!
this function should go to strhelpers.cpp / strhelpers.h

Reply:
`strAppendUnsigned(buf, val, 4)` does the same thing, it prepends by `0` though